scripts/patch-rpath

```diff
@@ -11,7 +11,7 @@ echo "Using final root directory of $root"
 
 cd $root
 
-for f in $(find bin lib jre -type f -executable -or -name "*.so*"); do
+for f in $(find bin lib jre -type f -executable -or -name "*.so*" | grep -v libc.so.6 | grep -v libpthread.so.0); do
   existing_rpath=$(patchelf --print-rpath $f 2>/dev/null || true)
-  patchelf --set-rpath '$ORIGIN/'$($root/bin/realpath --relative-to $(dirname $f) ${root%/}/lib):$existing_rpath $f 2>/dev/null || true
+  patchelf --set-rpath '$ORIGIN/'$(/usr/bin/realpath --relative-to $(dirname $f) ${root%/}/lib):$existing_rpath $f 2>/dev/null || true
```

Comment:
The docker image filesystem also serves as the standalone tar bundle that we distribute in a non-containerized form.  That is why we bundle realpath and refer to the bundled version here so that we aren't assuming it exists on the target system (it quite often doesn't).

What problem was the bundled realpath causing on ARM?